tagstudio/src/core/constants.py

```diff
@@ -86,8 +86,8 @@
 ]
 SPREADSHEET_TYPES: list[str] = [".csv", ".xls", ".xlsx", ".numbers", ".ods"]
 PRESENTATION_TYPES: list[str] = [".ppt", ".pptx", ".key", ".odp"]
-ARCHIVE_TYPES: list[str] = [".zip", ".rar", ".tar", ".tar", ".gz", ".tgz", ".7z"]
-PROGRAM_TYPES: list[str] = [".exe", ".app"]
+ARCHIVE_TYPES: list[str] = [".zip", ".rar", ".tar", ".tar", ".gz", ".tgz", ".7z", ".s7z", ".dmg"]
+PROGRAM_TYPES: list[str] = [".exe", ".app", ".jar", ".msi"]
```

Comment:
Mentioned changes + Ruff formatting
```suggestion
ARCHIVE_TYPES: list[str] = [
    ".zip",
    ".rar",
    ".tar",
    ".tar",
    ".gz",
    ".tgz",
    ".7z",
    ".s7z",
]
PROGRAM_TYPES: list[str] = [".exe", ".app"]
```